components/PoisList/Actions.vue

```diff
@@ -42,7 +39,7 @@ export default defineNuxtComponent({
   methods: {
     url(format: 'geojson' | 'csv'): string {
       return getPoiByCategoryIdUrl(
-        this.$vidoConfig(useRequestHeaders()),
+        useSiteStore().config || this.$vidoConfig(useRequestHeaders()),
```

Comment:
Same [issue](https://github.com/teritorio/vido/pull/181/files#r1508355775)

We could only use `config` from store once all the pages supports it.

Reply:
`this.$vidoConfig(useRequestHeaders()` should be removed.

Reply:
<s>this.$vidoConfig(useRequestHeaders() should be removed.</s>

Like the other one, should be keep, as `Action` not only used in list ?

Reply:
Nope, it's only used in PoisTable now !

Reply:
ok